### PR TITLE
dumb chembomb

### PR DESCRIPTION
**Pull request by zjdtmkhzt**
## About the PR 
With the new change to let activators deactivate they could be used to bypass the cooldown on artifact chembombs, so I am just preempting whatever nerd would have used that first.
Bombs now have a recharge_delay thing that can be set in general, used for chembombs and devastating explosive bombs.

Also, makes process on bombs return early if they are the type that detonates instantly, because the other variant made 0 sense and caused them to explode twice sometimes, lol.
## Why's this needed? 
I'm dumb and make terrible code.

### CODE REVIEW

**Comment by zjdtmkhzt** [comment]
Chembombs have a bit of a delay before exploding now, due to the machine loop taking some time .. which honestly might not be a bad idea, if I made it more consistent, I think.

File: code/obj/artifacts/artifact_objects/bombs.dm [hunk 1]
```diff
@@ -208,11 +210,14 @@ ABSTRACT_TYPE(/datum/artifact/bomb)
...
 	explode_delay = 0
+	alarm_initial = null
+	alarm_during = null
+	alarm_final = null
 	react_xray = list(5,65,20,11,"HOLLOW")
 	validtypes = list("ancient","martian","eldritch","precursor")
 	validtriggers = list(/datum/artifact_trigger/force,/datum/artifact_trigger/heat,/datum/artifact_trigger/carbon_touch)
 	var/payload_type = 0 // 0 for smoke, 1 for foam, 2 for propellant, 3 for just dumping fluids
-	var/recharge_delay = 600
+	recharge_delay = 600
```
Comment (BenLubar) [inline code/obj/artifacts/artifact_objects/bombs.dm:220]: may as well make this 10 MINUTES as well

Comment (zjdtmkhzt) [reply code/obj/artifacts/artifact_objects/bombs.dm:220]: true

Comment (zjdtmkhzt) [inline code/obj/artifacts/artifact_objects/bombs.dm:220]:
```suggestion
	recharge_delay = 10 MINUTES
```

File: code/obj/artifacts/artifact_objects/bombs.dm [hunk 2]
```diff
@@ -85,12 +85,13 @@ ABSTRACT_TYPE(/datum/artifact/bomb)
 
 			// actual boom
 			SPAWN_DBG(10 SECONDS)
-				if (src.activated)
+				if (O.loc && src.activated)
```
Comment (ZeWaka) [inline code/obj/artifacts/artifact_objects/bombs.dm:88]: i feel like checking .disposed is your intention

Comment (zjdtmkhzt) [reply code/obj/artifacts/artifact_objects/bombs.dm:88]: Ah, so that's how you're supposed to do that, thanks!^^

Comment (zjdtmkhzt) [inline code/obj/artifacts/artifact_objects/bombs.dm:88]:
```suggestion
				if (!O.disposed && src.activated)
```

**Review by pali6 — APPROVED** [review]

**Pull request merged**